src/table/pagination-table/components/body/TableBodyWrapper.tsx

```diff
@@ -61,8 +61,10 @@ function TableBodyWrapper({
           className="sn-table-row"
         >
           {columns.map((column, columnIndex) => {
-            const { id, align } = column;
+            const { id, textAlign } = column;
             const cell = row[id] as Cell;
+            const autoAlign = () => (cell.qNum && !Number.isNaN(+cell.qNum) ? 'right' : 'left');
+            const align = textAlign.auto ? autoAlign() : textAlign.align;
```

Comment:
This we also could do on a handle-data level, when we create `rows`. Unfortunately we have do do it in a different place for the virtual scroll. But we could reuse the logic.

Reply:
Yes, that is how the old table behaves
<img width="192" alt="Screenshot 2023-03-09 at 09 48 34" src="https://user-images.githubusercontent.com/4471489/224358180-1d5b22e1-1d6d-4476-aee7-7fc97d955258.png">